### DISCUSSION TITLE
what is anything in this? i feel like I've tried everything but done nothing

### DISCUSSION THREAD

**Discussion by loglot** [TodePond/Arroost#82] [Q&A]
here is what I figured out
clicking the big circle makes smaller circles
clicking the small circle flashes it white for a second
D gives me a unique cursor that does nothing
you can zoom 
you can pan

**Comment by TodePond** [comment]
it's not ready yet, the big and small circles are just for debugging. same with the D button. hold it down and click on something to see it's debug info in the console. when they flash white, it's because they're "firing" for one beat. this is gonna be a node wire language and it's a lot further along than it seems because all the internals are 100% done, and the UI engine is also 100% done. so now I'm making the different pieces of the 'standard library' of the library. more info will come in the coming weeks at https://www.patreon.com/todepond

**Comment by loglot** [comment]
I see!
